src/views/components/application-status.njk

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row govuk-!-margin-top-4">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Application status</h2>
+    <p class="govuk-body">Refresh the page to see updated information. If you exit this page, you can return to it by signing in to <a href="{{ Paths.ROOT_URI }}">the Apply to strike off and dissolve a company service</a>.</p>
+  </div>
+
+  {% set signatoryRows = [] %}
+  {% for signatory in signatories %}
+    {% set signatoryRows =
+      (
+        signatoryRows.push(
+          [
+            {
+              html: signatory.name if not signatory.on_behalf_name else '<div class="on_behalf_name_' + loop.index0 + '">' + signatory.on_behalf_name + '</div><div class="on_behalf_of_signatory_name_section_' + loop.index0 + '">signing on behalf of ' + signatory.name + '</div>',
+              attributes: { id: "name-" + loop.index0 }
```

Comment:
We will need to use signatory.officer_id when we click the edit link in future. Should we make all id's use the officer_id, or is loop.index0 ok? What do you think?

Reply:
Happy to change it to the officer ID since it's available. I only picked `loop.index0` as I wanted each row id/class unique

Reply:
Kept it as loop.index as it is easier to assert upon predictably.